### PR TITLE
Adding logo assets

### PR DESCRIPTION
**Pull request by portsmouth**
Adding here the finalized logo. This is almost identical to the one posted on Slack, except:

  - extremely small adjustment to the arrow head, to make it perfectly centered on and orthogonal to the stroke

  - SVG was optimized to remove redundant elements (e.g. gradients). Main logo is now only 10Kb. PNGs exported at 800dpi.
 

# Assets

openpbr-logo

openpbr-logo-text

openpbr-logo-glyph

# White paper

image

# README

image

### CODE REVIEW

**Comment by portsmouth** [comment]
An admin is needed to change the project icon to the OpenPBR glyph.

**Comment by portsmouth** [comment]
Some alternative layouts for the title and logo placement.

I like the second version the most, with the logo+text used for the title, and the subtitle giving the detailed "OpenPBR Surface specification" and the version/date.

```md
**OpenPBR Surface**
<center>*Specification v1.1, 2024-06-28*.</center>

<center>
   <a href="https://github.com/AcademySoftwareFoundation/OpenPBR" title="Redirect to GitHub page"><img src="images/logo/openpbr-logo.svg" alt="homepage" height="48" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
   &nbsp;&nbsp;   &nbsp;&nbsp;
   <a href="https://www.aswf.io/" title="Redirect to ASWF page"><img src="images/aswf-logo-black.svg" alt="homepage" height="36" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
</center>
```

image

```md
<center>
   <a href="https://github.com/AcademySoftwareFoundation/OpenPBR"><img src="images/logo/openpbr-logo-glyph.svg" alt="homepage" height="80" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
   &nbsp;
   <a href="https://github.com/AcademySoftwareFoundation/OpenPBR"><img src="images/logo/openpbr-logo-text.svg" alt="homepage" height="100" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
</center>

<style>bigger { font-style: italic; font-weight: normal; font-size: 115%;}</style>
<center><bigger><i>OpenPBR Surface specification v1.1, 2024-06-28</i></bigger></center>
```
image

**Comment by portsmouth** [comment]
In aea298251776e719e9da6fd0801cf79cec59a70c added simplified forms of the logo suitable for icons, as requested/discussed on [Slack](https://academysoftwarefdn.slack.com/archives/C06365EAQMP/p1725884797777319?thread_ts=1715203811.853709&cid=C06365EAQMP).

image
image

**Comment by portsmouth** [comment]
In 54474ff8268725f84b852e027c29e7b755326beb updated the spec to use the logo (glyph next to text form) for the title, with the subtitle giving the full "OpenPBR Surface" (+version/date) wording.

**Comment by portsmouth** [comment]
Added proposed 128x128 favicon in 95cc4baf477684eeedcf3f5657078b861b58271e

image

File: index.html [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 <link rel="stylesheet" href="style/style.css">
+<link rel="shortcut icon" type="image/png" href="images/logo/openpbr-logo-favicon.png">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 
-**OpenPBR Surface**
+<center>
+   <a href="https://github.com/AcademySoftwareFoundation/OpenPBR"><img src="images/logo/openpbr-logo-glyph.svg" alt="homepage" height="80" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
+   &nbsp;
+   <a href="https://github.com/AcademySoftwareFoundation/OpenPBR"><img src="images/logo/openpbr-logo-text.svg" alt="homepage" height="100" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
+</center>
```
Comment (virtualzavie) [inline index.html:10]: As a minor remark, I think we lost a bit of semantic here by replacing the header text with an image, but I don't know what would be the correct way of doing it within Markdeep.

**Review by virtualzavie — APPROVED** [review]
Thank you! I'm really pleased to see how good this looks both on dark and light backgrounds.

File: index.html (hunk 1 above)
Comment (portsmouth) [reply index.html:10]: Yeah, though I'm not sure it matters that the Markdeep title is missing. I would have preferred to have the subtitle be the Markdeep title, and put the logo above, but Markdeep doesn't seem to allow that.

**Review by AdrienHerubel — APPROVED** [review]
New logo and updated specification LGTM

**Comment by meshula** [comment]
looking good!

**Review by jstone-lucasfilm — APPROVED** [review]
These changes look good to me, thanks @portsmouth!

**Pull request merged**